Gemfile.lock

```diff
@@ -745,6 +745,7 @@ PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
   arm64-darwin-23
+  arm64-darwin-24
```

Comment:
this started showing up in when i last did a `db:migrate`, so i am adding this in here even though it doesn't have to do with this ticket.

Reply:
Yeah, it just means your OS version is slightly different than others.  These tend to just show up over time, as you bundle-install.